libs/coin-framework/src/network.ts

```diff
@@ -0,0 +1,130 @@
+import invariant from "invariant";
```

Comment:
This file is currently used in a few contexts other than coin-integration, but not many, so either we could rework these places to stop using it, or it should be extracted in its own lib/util, right?
In either case a `TODO` comment can be put here, and we should add it to the list of remaining tasks.